wp-content/themes/flightfindertheme/db_query.php

```diff
@@ -0,0 +1,40 @@
+<?php
+		
+		include('db_connect.php');
+		
+		if (isset($_REQUEST['origin'])) {
+			$origin_airport = $_REQUEST['origin'];
+			$sql_res = mysql_query("select id,iata,city,full_name,latitude,longitude from airports where full_name like '$origin_airport%' or city like '$origin_airport%' or iata like '$origin_airport%' order by full_name ASC");
```

Comment:
mysql_query is deprecated and you do not need to use it. Use
`$wpdb->get_results($wpdb->prepare( $sql, $param1, $param2))`

https://developer.wordpress.org/reference/classes/wpdb/prepare/